media/js/mozorg/home/home-a.js

```diff
@@ -115,6 +116,25 @@
             handler: navState('firefox', 'innovation'),
             offset: '30%'
         });
+
+        surveyWaypoint = new Waypoint({
+            element: '#firefox',
+            handler: function(direction) {
+                var $surveyMsg = $('#survey-message');
+                if ($surveyMsg.length) {
+                    if (direction === 'down') {
+                        // slide up when scrolling down
+                        $surveyMsg.addClass('stuck').css({ bottom: '-' + $surveyMsg.height() + 'px' }).animate({ bottom: '0' }, 500);
+                    } else if (direction === 'up') {
+                        // slide down when scrolling up, then unstick
+                        $surveyMsg.animate({ bottom: '-' + $surveyMsg.height() + 'px' }, 500, function() {
+                            $surveyMsg.removeClass('stuck').removeAttr('style');
```

Comment:
As removing the `stuck` class reverts to `display: none;`, I don't think you need to worry about removing the `style` attribute.